tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -24,9 +24,9 @@ When(
     const resourceObject = new objects.applicationFiles.Resource({ page })
     for (const info of stepTable.hashes()) {
       await resourceObject.upload({
-        to: info.to !== 'undefined' ? info.to : null,
+        to: info.to !== undefined ? info.to : null,
```

Comment:
@SwikritiT is it okey to remove the quotes?

Reply:
if the tests are passing I guess it's okay